src/watertap_contrib/seto/unit_models/surrogate/tests/test_lt_med_surrogate.py

```diff
@@ -370,7 +379,7 @@ def test_interp_values(self, number_effects):
             "flow_mass_phase_comp", 1e-2, index=("Liq", "H2O")
         )
         m.fs.liquid_prop.set_default_scaling(
-            "flow_mass_phase_comp", 1e3, index=("Liq", "TDS")
```

Comment:
I think this was the source of some of your scaling problems, @Zhuoran29 . After revising this, I didn't need constraint scaling anymore at all, and the issue with having to set specific area value for 10 effects during initialization was resolved too.

Reply:
Thanks for identifying the issue! It makes sense just by looking at it. I'll test and do the same for MED-TVC, then commit to the same PR. Will update this comment after I do so.

Reply:
Actually, I think it's better to just do on a separate PR.

Reply:
Ok, sure.